Packs/PANOSPolicyOptimizer/Integrations/PANOSPolicyOptimizer/README.md

```diff
@@ -1,5 +1,7 @@
 Automate your AppID Adoption by using this integration together with your Palo Alto Networks Next-Generation Firewall or Panorama.
 This integration was integrated and tested with version 8 up to 10.1.6 of PAN-OS Policy Optimizer, later versions may not work
```

Comment:
```suggestion
This integration was integrated and tested with version 8 up to 10.1.6 of PAN-OS Policy Optimizer, later versions may not work.


```
Also please fix the table of create instances before merging

Reply:
ok